browserscripts/browser/collectStats.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
```

Comment:
nit: call the file `perfStats` (or `privilegedPerfStats`), for greppability and to follow suite with the existing measurement files, which are called like what they return (`userAgent.js`), not what they are doing (`collectUserAgent.js`).

Reply:
Since we are not committing this file in this revision, I'm going to simply mark this as resolved.